bin/drush-install.sh

```diff
@@ -35,6 +35,22 @@ org.civicrm.contactsummary,\
 org.civicrm.bootstrapcivicrm,\
 org.civicrm.bootstrapcivihr
 
+##
+# Set Default localisation settings
```

Comment:
I don't know how common it is in bash scripts, but you're clearly using a parameter in this function (`${CIVI_CORE}` from [here](https://github.com/civicrm/civicrm-buildkit/pull/274/files#diff-f450058aa61268db75c9dc084fcefee8R46)) and it would be much clearer if you could document it in the function what that `$1` is and what it is for